### PR TITLE
Add wallet recovery public api key to missions

### PR DESCRIPTION
**Pull request by Lerek**

### CODE REVIEW

**Comment by Riddlerrr** [comment]
I think the `wallet_recovery_api_public_key` should be available to set using API only and not available to show/edit using UI.
But let's check with @aquabu

**Comment by aquabu** [comment]
This is the public key used for signing API requests similar to the Whitelabel API key.

You should be able to add it from the missions page. But it should not be editable after it is posted. Currently you would need to contact a site administrator to change it after it has been posted.

I don't think there is an API endpoint for this and there probably shouldn't be for now.

File: app/javascript/components/MissionForm.jsx [hunk 1]
```diff
@@ -365,6 +397,14 @@ export default class MissionForm extends React.Component {
             eventHandler={this.handleChangeFormData}
           />
 
+          <InputFieldWhiteDark
+          name='walletRecoveryApiPublicKey'
+          title='Wallet recovery API Public Key'
+          symbolLimit={68}
+          value={walletRecoveryApiPublicKey}
```
Comment (Alexey1100) [inline app/javascript/components/MissionForm.jsx:404]:
```suggestion
          value={walletRecoveryApiPublicKey}
          readOnly={this.props.mission.walletRecoveryApiPublicKey}
```

**Review by Alexey1100 — CHANGES_REQUESTED** [review]
Should be read-only once set

File: app/javascript/components/MissionForm.jsx (hunk 1 above)
Comment (aquabu) [reply app/javascript/components/MissionForm.jsx:404]: Yes and the read only property should be on the model level so the form can't be bypassed.

**Comment by Lerek** [comment]
@aquabu , @Alexey1100 done, could you take a look again, please?

**Review by Riddlerrr — CHANGES_REQUESTED** [review]
Let'a add `wallet_recovery_api_public_key` as a readonly attribute for the model. Like we do for example here:
https://github.com/CoMakery/comakery-server/blob/b7aa5316cd70b03051592d841543fee26defcf9a/app/models/reg_group.rb#L15

**Comment by Lerek** [comment]
> Let'a add `wallet_recovery_api_public_key` as a readonly attribute for the model. Like we do for example here:
> https://github.com/CoMakery/comakery-server/blob/b7aa5316cd70b03051592d841543fee26defcf9a/app/models/reg_group.rb#L15

@Riddlerrr done

**Review by Riddlerrr — APPROVED** [review]

**Comment by aquabu** [comment]
@Lerek I think this issues are fixed but there's a merge conflict.

**Comment by Lerek** [comment]
@aquabu done)

**Review by Alexey1100 — APPROVED** [review]

**Pull request merged**